package.json

```
@@ -7,6 +7,7 @@
     "test:defaultPath": "\"./node_modules/.bin/cucumber-js\" -p default \"node_modules/openapi-forge/features/*.feature\" \"openapi-forge/src/generate\"",
     "format:check": "prettier --check .",
     "format:write": "prettier --write .",
+    "format:output": "prettier --write",
```

I am wondering if we should change the naming convention:
For the existing commands add a ':all' after as it targets all the files. And for your new command use 'format:write' ?

What do you think?

```suggestion
    "format:check:all": "prettier --check .",
    "format:write:all": "prettier --write .",
    "format:write": "prettier --write",
    "lint:check:all": "eslint .",
    "lint:write:all": "eslint --fix ."
```